samples/Cosmonaut.NetCore2WebApi/Controllers/SampleController.cs

```diff
@@ -0,0 +1,169 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Cosmonaut.Extensions;
+using Cosmonaut.NetCore2WebApi.Contract;
+using Cosmonaut.NetCore2WebApi.Model;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Azure.Documents.SystemFunctions;
+using Microsoft.EntityFrameworkCore;
+
+namespace Cosmonaut.NetCore2WebApi.Controllers
+{
+    public class SampleController : Controller
+    {
+        private readonly ICosmosStore<Person> _personsCosmosStore;
+
+        public SampleController(ICosmosStore<Person> personsCosmosStore)
+        {
+            _personsCosmosStore = personsCosmosStore;
+        }
+
+        [HttpGet("/api/persons")]
+        public async Task<IActionResult> GetAllPaged([FromQuery] int pageSize, [FromQuery] string continuationToken,
+            CancellationToken cancellationToken)
+        {
+            if (pageSize > 50)
+            {
+                return BadRequest("Page Size too big! Must be < 50");
+            }
+
+            if (continuationToken == null || continuationToken.Trim().Length == 0)
+            {
+                var initialPage = await _personsCosmosStore.Query().OrderBy(person => person.Name)
+                    .WithPagination(0, pageSize).ToPagedListAsync(cancellationToken: cancellationToken);
```

Comment:
This will throw an exception as `pageNumber` is 0 which is not a valid page number.